lib/smart_answer_flows/register-a-birth/outcomes/oru_result.govspeak.erb

```diff
@@ -215,13 +215,20 @@
 
   <% if calculator.country_of_birth == 'venezuela' && calculator.same_country? %>
```

Comment:
Does this true case of the `if` now do the same as the `else`? Can we get rid of the `if` statement completely?

Reply:
`same_country?` and `in_the_uk?` are mutually exclusive, so yes it is.